### PR TITLE
new main template

### PR DESCRIPTION
**Pull request by Sadat97**
I didn't test the input of the character

### CODE REVIEW

File: src/Team17/Main.java [hunk 1]
```diff
@@ -1,76 +1,135 @@
...
-		 {
-			 //Add your function here also
-			 Count_Primes obj = null;
-			 System.out.println("The number of prime numbers:"+obj.Count_Primes(arr));
-		 }
-			 break;
-		default:
-		System.out.println("Invalid input!");
-		break;
+		System.out.println("Press 1 to input numbers and 2 for characters");
```
Comment (MaryamOsamaX) [inline src/Team17/Main.java]: What if he/she entered 1 and input chars? we could check by ascii instead or if you have another solution

File: src/Team17/Main.java [hunk 2]
```diff
@@ -1,76 +1,135 @@
...
+
+		if (typeOfArray == 1) {
+			String[] numbers = array.replace(" ", "").split(",");
+			int[] a = new int[numbers.length];
+			for (int i = 0; i < numbers.length; i++) {
+				System.out.println(numbers[i]);
+				a[i] = Integer.parseInt(numbers[i]);
+			}
+		} else {
+			char[] char_array = array.toCharArray(); //i haven't test this well
```
Comment (MaryamOsamaX) [inline src/Team17/Main.java]: it works correctly

File: src/Team17/Main.java [hunk 3]
```diff
@@ -1,76 +1,135 @@
...
+				System.out.println("hekko");
+				switch_runner(choice);
+			}
+			print_messages();
+			 choice = input.nextInt();
+		}
+		
+	}
+
+	private static void print_messages(){
```
Comment (MaryamOsamaX) [inline src/Team17/Main.java]: I think the function should be above the main

File: src/Team17/Main.java [hunk 4]
```diff
@@ -1,76 +1,135 @@
...
+		System.out.println("<<<Main Menu>>>");
+		System.out.println("1-Most repeated value\t2-Sort\t3-Shuffle\t4-Find the largest prime\t5-Find the smallest prime\t6-Check palindrome");
+		System.out.println("7-Check sorted\t8-Count Primes-\t9-Reverse array\t10-Shift array\t11-Distinct array\t12-Get the maximum 3 numbers");
+		System.out.println("13-Get the minimum 3 numbers\t14-Get average\t15-Get median\t16-Return only primes\t17-Zero if less than zero\t18-to execute all");
+		System.out.println("press 0('zero') to exit");
+		System.out.println("Enter the number of which function to be executed:");
 	}
+	private static void switch_runner(int choice){
```
Comment (MaryamOsamaX) [inline src/Team17/Main.java]: I think the function should be above the main

File: src/Team17/Main.java (hunk 3 above)
Comment (Sadat97) [reply src/Team17/Main.java]: what's the difference as i test it and it's working

**Pull request merged**